### ISSUE TITLE
Can I lower down the ddim sample steps in the training?

### ISSUE THREAD

**Issue by Zhaoyi-Yan** [lllyasviel/ControlNet#421]
It seems unnecessary to perform ddim sample during training and it is slow.

**Comment by engrmusawarali** [comment]
Yes you can lower down. Just go to the logger in cldm and  you can modify 

 if not self.disabled and pl_module.current_epoch % 10 == 0:
